openlr/road_type_checkers.cpp

```diff
@@ -0,0 +1,56 @@
+#include "openlr/road_type_checkers.hpp"
+
+#include "indexer/classificator.hpp"
+
+namespace openlr
+{
+// TrunkChecker ------------------------------------------------------------------------------------
+TrunkChecker::TrunkChecker()
```

Comment:
We'll never find such checkers in "openlr/road_type_checkers.cpp" but other parts of the project can be interested in them. I consider such checker as general functionality which can be useful for turn generation system as well. I'm afraid keeping them here there's a big chance to reimplement them.